test/modules/layers/test_quantisation.py

```diff
@@ -105,8 +78,47 @@ def test_quantised_shape(self):
         vq.embedding = nn.Embedding.from_pretrained(self.embedding_weights)
         output = vq(self.encoded)
```

Comment:
I thought that pytest doesn't keep any variables across different tests, so I repeated the forward pass. or maybe I need to keep it in a class variable? also not sure if I even need `test_quantised_shape` since you mentioned the shapes are already compared in `test_quantised_output`